src/main/scala/cromwell/authentication/oauth2/ClientSecrets.scala

```diff
@@ -0,0 +1,8 @@
+package cromwell.authentication.oauth2
+
+trait ClientSecrets {
+  val clientId: String
```

Comment:
abstract members of traits should generally be defs as you can always go down the hierarchy. In theory there'd be an implementer where it doesn't make sense for that to be a val and you can't go upwards